packages/perseus/src/components/svg-image.tsx

```diff
@@ -485,12 +485,22 @@ class SvgImage extends React.Component<Props, State> {
                 // without coordinates. They don't seem to have any content, so
                 // it seems fine to just ignore them (rather than error), but
                 // we should figure out why this is happening.
+
+                // 'fontSize' - When a default scale (1) is used,
+                //     setting the font size to 100% is redundant.
+                // Also, setting the font size to 100% can counteract other scale-based styling.
+                // Therefore, when the scale is not applicable,
+                //     we set the value to 'null' so that it won't be set to anything.
+                const fontSize =
```

Comment:
I think we'd be slightly more clear calling this variable `style` as that's what the last parameter is (extra styles).  If we call this `fontSize` here, I could see folks being confused how to integrate other custom styles. 

Also, I realize this is ancient code that we probably won't touch much before deleting it (oh what a glorious day that would be)... but I think updating the variable name could still avoid future confusion.

Reply:
Done.